File: CS/GettingStarted3/Models/Gdp.cs
namespace GettingStarted3.Models {
    public class Gdp {
        public string Country { get; private set; }
        public int Year { get; private set; }
        public double Product { get; private set; }

        public Gdp(string country, int year, double product) {
            this.Country = country;
            this.Year = year;
            this.Product = product;
        }
    }
}

File: VB/GettingStarted3/Models/Gdp.vb
Public Class Gdp
    Private mProduct As Double
    Private mYear As Integer
    Private mCountry As String

    Public ReadOnly Property Country() As String
        Get
            Return mCountry
        End Get
    End Property
    Public ReadOnly Property Year() As Integer
        Get
            Return mYear
        End Get
    End Property
    Public ReadOnly Property Product() As Double
        Get
            Return mProduct
        End Get
    End Property

    Public Sub New(country As String, year As Integer, product As Double)
        mCountry = country
        mYear = year
        mProduct = product
    End Sub
End Class